package.json

```diff
@@ -71,9 +72,9 @@
     "build": "npm run build-js && npm run build-css",
     "watch": "npm run build-example && npm run watch-example && (budo dist/example/main.js --live --css dist/example/main.css)",
     "lint": "eslint src",
-    "execute-tests": "run-browser-babel dist/test/index.js --phantom",
+    "execute-tests": "run-browser-babel dist/test/index.js --bx react/addons --bx react/lib/ReactContext --bx react/lib/ExecutionEnvironment --phantom",
```

Comment:
There's an `if (REACT013)`, at the beginning of the block. Perhaps it's a bug in enzyme?